tests/readers/cyme/test_connection_configuration_mapping.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from ditto.readers.cyme.read import Reader
+from itertools import chain
+
+@pytest.mark.parametrize('value, expected_configuration', list(chain(
+    [(el, 'Y') for el in (0, "0", "Yg", "yg")],
```

Comment:
I wanted to have a 1-1 mapping from the test parameters to the table that is the [docstring for the function](https://github.com/NREL/ditto/blob/db6767aefc73c7d53a910f6e56d6a72dc86d378a/ditto/readers/cyme/read.py#L536), so it would be easier to compare for completeness.